java/client/src/org/openqa/selenium/support/ui/FluentWait.java

```diff
@@ -68,14 +69,18 @@
  */
 public class FluentWait<T> implements Wait<T> {
 
-  public static final Duration FIVE_HUNDRED_MILLIS = new Duration(500, MILLISECONDS);
+  public final static long DEFAULT_SLEEP_TIMEOUT = 500;
+
+  public static final Duration FIVE_HUNDRED_MILLIS = new Duration(DEFAULT_SLEEP_TIMEOUT, MILLISECONDS);
```

Comment:
We should mark this field deprecated. It's used once by another part of the selenium codebase, and is a trivial constant for others to declare.